tests/test_qpex.py

```diff
@@ -105,6 +107,14 @@ def create_qps(self):
             self.qps_num.append((recv_qp.qp_num, send_qp.qp_num))
             self.psns.append(random.getrandbits(24))
 
+class QpExRCRDMAFlush(RCResources):
+    type = e.IB_FLUSH_GLOBAL
+    level = e.IB_FLUSH_MR
+    def create_qps(self):
+        create_qp_ex(self, e.IBV_QPT_RC, e.IBV_QP_EX_WITH_RDMA_FLUSH | e.IBV_QP_EX_WITH_RDMA_WRITE)
+
+    def create_mr(self):
+        self.mr = u.create_custom_mr(self, e.IBV_ACCESS_FLUSH_GLOBAL | e.IBV_ACCESS_REMOTE_WRITE)
```

Comment:
Do you mean ibv_reg_mr(3) should return "not supported" ?  The current implementation in the kernel will return *[EOPNOTSUPP](https://github.com/torvalds/linux/blob/208e3a134b50d95ea3962d7a37b4d8a8f5368376/include/rdma/ib_verbs.h#L4356)* in case of non-use devices.

and the tests tell this too.
```
[root@iaas-rpma build]# rdma link
link rocep1s0f0/1 state ACTIVE physical_state LINK_UP netdev ens1f0np0
link rocep1s0f1/1 state DOWN physical_state DISABLED netdev ens1f1np1
link rxe_eno1/1 state ACTIVE physical_state LINK_UP netdev eno1
[root@iaas-rpma build]# ./bin/run_tests.py --dev rocep1s0f0 --gid 1 -v tests.t
est_qpex.QpExTestCase.test_qp_ex_rc_rdma_flush tests.test_qpex.QpExTestCase.te
st_qp_ex_rc_rdma_flush_type_violate
test_qp_ex_rc_rdma_flush (tests.test_qpex.QpExTestCase) ... skipped 'Create custom mr with additional access flags 258 is not supported'
test_qp_ex_rc_rdma_flush_type_violate (tests.test_qpex.QpExTestCase) ... skipped 'Create custom mr with additional access flags 258 is not supported'

----------------------------------------------------------------------
Ran 2 tests in 0.025s

OK (skipped=2)
[root@iaas-rpma build]# ./bin/run_tests.py --dev rxe_eno1 --gid 1 -v tests.test_qpex.QpExTestCase.test_qp_ex_rc_rdma_flush tests.test_qpex.QpExTestCase.test_qp_ex_rc_rdma_flush_type_violate
test_qp_ex_rc_rdma_flush (tests.test_qpex.QpExTestCase) ... ok
test_qp_ex_rc_rdma_flush_type_violate (tests.test_qpex.QpExTestCase) ... ok

----------------------------------------------------------------------
Ran 2 tests in 0.018s

OK
```